src/hotspot/share/interpreter/zero/bytecodeInterpreter.cpp

```diff
@@ -1372,7 +1373,10 @@ void BytecodeInterpreter::run(interpreterState istate) {
 #define ARRAY_INTRO(arrayOff)                                                  \
       arrayOop arrObj = (arrayOop)STACK_OBJECT(arrayOff);                      \
       jint     index  = STACK_INT(arrayOff + 1);                               \
-      char message[2 * jintAsStringSize + 35];                                 \
+      const int add_len = 32;                                                  \
+      STATIC_ASSERT(add_len == strlen("Index  out of bounds for length "));    \
```

Comment:
The double-space there is a bit subtle. I'm not sure static_asserting the length of a different string literal really aids the reader here at all. A simple comment would have sufficed in my view.

Reply:
I think it captures the intent well enough. The code in `Runtime1` uses a similar assert, and it helped me to get the gist of the "35" as the length. So, I prefer to static assert it here.